core/components/molecules/stepper/__stories__/variants/StepperWithTrigger.story.jsx

```diff
@@ -37,7 +35,14 @@ export const stepperWithTrigger = () => {
   );
 };
 
-const customCode = `() => {
+const customCode = `/*
+// style.css
+.Stepper-body {
+    height: 200px;
+}
```

Comment:
Instead of using fixed height please use `pt-11` classname on Button wrapper div